### PR TITLE
Support shutdown

### PR DESCRIPTION
**Pull request by richardschneider**
See issue https://github.com/ipfs/js-ipfs/issues/1192

### CODE REVIEW

File: src/utils/load-commands.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ function requireCommands () {
     ping: require('../ping'),
     refs: require('../refs'),
     repo: require('../repo'),
+    shutdown: require('../shutdown'),
```
[the remark this answers is not in the document]
Comment (richardschneider) [reply src/utils/load-commands.js]: Well `go-ipfs` has implemented `ipfs shutdown`.  It's the reference standard.

[the remark this answers is not in the document]
Comment (richardschneider) [reply src/utils/load-commands.js]: https://ipfs.io/docs/api/#apiv0shutdown

**Comment by richardschneider** [comment]
@diasdavid RTM

**Comment by diasdavid** [comment]
released `interface-ipfs-core@0.46.0` with the tests for this PR

**Pull request merged**